homeassistant/components/jellyfin/__init__.py

```diff
@@ -15,7 +15,7 @@ async def async_setup_entry(hass: HomeAssistant, entry: ConfigEntry) -> bool:
     """Set up Jellyfin from a config entry."""
     hass.data.setdefault(DOMAIN, {})
 
-    client = create_client()
+    client = create_client(entry.unique_id)  # type: ignore[arg-type]
```

Comment:
Is this ignore really necessary?

Reply:
No, it wasn't :) Fixed that with an additional check.